app/src/main/java/com/example/ithaca_transit_android_v2/NetworkingAdapters.kt

```diff
@@ -44,4 +46,128 @@ class LocationAdapter {
         }
         return DataLocation(final)
     }
+}
+
+@JsonClass(generateAdapter = true)
+class RouteAdapter {
```

Comment:
this adapter includes both the decoding of `Route` and `Direction` and the `Direction` decoding relies on the decoding of `Location`. you might be able to remove the code for decoding `Direction` if you add the `LocationAdapter()`  to moshi above in `NetworkUtils`.